app.py

```diff
@@ -10,7 +9,8 @@
 def index():
     item_list = []
     list_id = '5f56323626c33d81cd98b386'
```

Comment:
We shouldn't be hardcoded any list ids here. Any list ids should be moved over to the `.env` file and mentioned in `.env.template` so that new developers with their own boards will no what they need to add.

Also `.env.template` is missing `KEY` & `TOKEN` entries.